src/main/java/fr/skyost/owngarden/OwnGarden.java

```diff
@@ -0,0 +1,166 @@
+package fr.skyost.owngarden;
+
+import com.google.common.base.Joiner;
+import fr.skyost.owngarden.command.OwnGardenCommand;
+import fr.skyost.owngarden.data.MaterialSchems;
+import fr.skyost.owngarden.listener.GlobalEventsListener;
+import fr.skyost.owngarden.util.ZipUtils;
+import fr.skyost.owngarden.worldedit.DefaultUtils;
+import fr.skyost.owngarden.worldedit.Utils;
+import fr.skyost.owngarden.worldedit.WorldEditUtils;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.logger.slf4j.ComponentLogger;
+import net.kyori.adventure.text.minimessage.MiniMessage;
+import net.kyori.adventure.text.minimessage.tag.resolver.TagResolver;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.serialization.ConfigurationSerialization;
+import org.bukkit.plugin.java.JavaPlugin;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.SecureRandom;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.function.Consumer;
+
+/**
+ * The OwnGarden plugin class.
+ */
+
+public class OwnGarden extends JavaPlugin {
+
+    public static final String TREES = "trees";
+
+    public static final MiniMessage mini = MiniMessage.builder()
+        .tags(TagResolver.builder().resolver(TagResolver.standard())
+            .build()).build();
```

Comment:
You dont need this, because `MiniMessage.miniMessage().deserialize("your string")` does the same